creator-node/src/services/prometheusMonitoring/prometheusRegistry.js

```diff
@@ -0,0 +1,46 @@
+const PrometheusClient = require('prom-client')
+
+const {
+  METRIC_PREFIX,
+  Metrics,
+  MetricNames
+} = require('./prometheus.constants')
+
+module.exports = class PrometheusRegistry {
+  constructor() {
+    this.registry = PrometheusClient.register
+    this.metricNames = MetricNames
+
+    // Enable collection of default metrics (e.g. heap, cpu, event loop)
+    PrometheusClient.collectDefaultMetrics({ prefix: METRIC_PREFIX })
```

Comment:
Maybe `audius_cn_promclient_`? I'm not in love with `promclient`, but just something to separate our namespace from theirs. Even if it's not an issue today, that would prevent new metrics added via `collectDefaultMetrics` from clashing with our own metrics.

Example: If `collectDefaultMetrics` produces a metric like `storage`, then it would harder for us to differentiate if that was our `audius_cn_storage_*` or their `audius_cn_storage_*`.

Also, would you mind sharing some exporter output so we can see the format these default metrics are in?

Reply:
good point - will add a custom namespace to these

example exporter output is in the PR description! pasted here for convenience
![image](https://user-images.githubusercontent.com/3323835/173349057-9ef3cc95-fcf5-4da3-bfe8-8be3157b71a5.png)

Reply:
Oh, sorry for not specifying, but I was primarily interested in the `collectDefaultMetrics` exporter output and I did find a snippet in the PR description as well, thanks!

And sounds good on the custom namespace, thanks!!

Reply:
ah gotcha, sweet!